content/en/news/releases/1.20.x/announcing-1.20/change-notes/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Istio 1.20.0 Change Notes
+linktitle: 1.20.0
+subtitle: Minor Release
+description: Istio 1.20.0 change notes.
+publishdate: 2023-11-14
+release: 1.20.0
+weight: 20
+---
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according
+to [Istio's deprecation policy](/docs/releases/feature-stages/#feature-phase-definitions). Please consider upgrading
+your
+environment to remove the deprecated functionality.
+
+- There are no new deprecations in Istio 1.20.0
+
+## Traffic Management
+
+- **Improved** support for `ExternalName` services. See the Upgrade Notes for more information.
+
+- **Improved** the ordering of HTTP and TCP envoy filters to enhance consistency.
+
+- **Improved** `iptables` locking. The new implementation uses the built-in `iptables` lock waiting when needed, and
+  disables locking entirely when it's not needed.
+
+- **Improved** `WorkloadEntry` resources added inline via the `endpoints` field in `ServiceEntry` resources on different
+  networks to no longer require an address to be specified.
+  ([Issue #45150](https://github.com/istio/istio/issues/45150))
+
+- **Added** support for traffic mirroring to multiple destinations in `VirtualService`.
+  ([Issue #13330](https://github.com/istio/istio/issues/13330))
+
+- **Added** the ability for the user to specify the `ipFamilyPolicy` and `ipFamilies` settings in Istio Service
+  resources either via the operator API or the Helm charts.
+  ([Issue #44017](https://github.com/istio/istio/issues/44017))
+
+- **Added** support for network `WasmPlugin`.
+
+- **Added** a gated flag `ISTIO_ENABLE_IPV4_OUTBOUND_LISTENER_FOR_IPV6_CLUSTERS` to manage an additional outbound
+  listener
+  for IPv6-only clusters to handle IPv4 NAT outbound traffic.
+  This is useful for IPv6-only cluster environments such as EKS, which manages both egress-only IPv4 and IPv6 IPs.
+  ([Issue #46719](https://github.com/istio/istio/issues/46719))
+
+- **Added** the capability to attach `AuthorizationPolicy` to Kubernetes `Gateway` resources via the `targetRef`
+  field. ([Issue #46847](https://github.com/istio/istio/issues/46847))
+
+- **Added** `failoverPriority` and `failover` to work in conjunction with each other.
```

Comment:
I'm not sure Added is the correct verb here. Maybe Updated with the corresponding move.

Reply:
Sounds good